extension/lib/doc.js

```diff
@@ -70,9 +73,9 @@ class Doc {
         //   first sentence?
         // - $list [] sentenceEls. See getSentenceEls().
         this.detectSentenceBoundaries(this.containers);
-        this.termFreq = null; // { str : int } , Frequency of terms in document. Set in calcTotalWords function
-        this.total_words = this.calcTotalWords(readableDomEls); // int; Total number of words in document;
-        this.keywords = this.setKeyWords(this.termFreq); // string[] keywords of document
+        this.termFreq = null; // { str : int } , Frequency of terms in document. Set in updateWordModel function
+        this.total_words = null; // int; Total number of words in document;
+        this.keywords = null; // string[] keywords of document
```

Comment:
there's a line below "this.num_docs = null" that is no longer used.
pls remove

-- Can't comment on non-edited line :(

Reply:
damn damn damn 
👍